### PR TITLE
Handle card labels

### PR DESCRIPTION
**Pull request by dobbymoodge**
This is to address https://github.com/jeremytregunna/ruby-trello/issues/199

I rewrote the PR from scratch and think I've found the best solution. The tests will need to be updated accordingly.

@joneslee85

### CODE REVIEW

File: lib/trello/card.rb [hunk 1]
```diff
@@ -173,6 +178,7 @@ def update_fields(fields)
       attributes[:member_ids]             = fields[SYMBOL_TO_STRING[:member_ids]]
       attributes[:list_id]                = fields[SYMBOL_TO_STRING[:list_id]]
       attributes[:pos]                    = fields[SYMBOL_TO_STRING[:pos]]
+      attributes[:labels]                 = (fields[SYMBOL_TO_STRING[:labels]] || []).map{ |lbl| Trello::Label.new(lbl) }
```
Comment (joneslee85) [inline lib/trello/card.rb]: nitpick, a space before `{`

**Comment by joneslee85** [comment]
:+1: LGTM

cc @jeremytregunna

**Comment by jeremytregunna** [comment]
@dobbymoodge This is good to go now?

**Comment by dobbymoodge** [comment]
@jeremytregunna let me handle the one nit @joneslee85 pointed out, and squash the commits

**Comment by dobbymoodge** [comment]
@jeremytregunna OK, good to merge now

**Comment by jeremytregunna** [comment]
Ok when the tests finish, let's get this merged then.

**Comment by joneslee85** [comment]
thanks @dobbymoodge

**Pull request merged**